deviceplugin/pkg/instrumentation/lister.go

```diff
@@ -48,22 +46,15 @@ func (l *lister) NewPlugin(s string) dpm.PluginInterface {
 // Otel SDKs frequently requires to set some environment variables and mount some fs dirs for it to work.
 type OtelSdksLsf map[common.ProgrammingLanguage]map[common.OtelSdk]LangSpecificFunc
```

Comment:
note: this is still being used in enterprise thus not removed completely.
will be removed once we retire dotnet-legacy hopefully soon